torchok/constructor/logger.py

```diff
@@ -1,19 +1,46 @@
+import os
+from argparse import Namespace
 from itertools import chain
 from pathlib import Path
 from typing import Any, Dict, Optional, Union, MutableMapping
 
-from pytorch_lightning.loggers.logger import Logger
+from omegaconf.listconfig import ListConfig
 from pytorch_lightning.loggers.csv_logs import CSVLogger
+from pytorch_lightning.loggers.logger import Logger
 from pytorch_lightning.loggers.mlflow import MLFlowLogger
 from pytorch_lightning.loggers.mlflow import rank_zero_only
-from pytorch_lightning.utilities import rank_zero_warn
 from pytorch_lightning.loggers.neptune import NeptuneLogger
 from pytorch_lightning.loggers.tensorboard import TensorBoardLogger
 from pytorch_lightning.loggers.wandb import WandbLogger
+from pytorch_lightning.utilities import rank_zero_warn
 from pytorch_lightning.utilities.logger import _convert_params
 
-from argparse import Namespace
-from omegaconf.listconfig import ListConfig
+
+def create_logger(logger_config) -> Logger:
```

Comment:
can you please write documentation for this function

Reply:
Added